tools/ibc-solidity-abi/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod generated;
```

Comment:
Is it possible to use https://docs.rs/ethers/latest/ethers/middleware/contract/macro.abigen.html?